### PR TITLE
Cleanup normalize

### PR DESCRIPTION
**Pull request by serprex**
Clean up code, fetch `syncBatchID`/`normalizeBatchID` together

Reduce redundantly setting FlowName on context in cdc_flow

### CODE REVIEW

File: flow/connectors/bigquery/bigquery.go [hunk 1]
```diff
@@ -382,29 +382,32 @@ func (c *BigQueryConnector) GetLastSyncBatchID(jobName string) (int64, error) {
 	}
 }
 
-func (c *BigQueryConnector) GetLastNormalizeBatchID(jobName string) (int64, error) {
-	query := fmt.Sprintf("SELECT normalize_batch_id FROM %s.%s WHERE mirror_job_name = '%s'",
+func (c *BigQueryConnector) GetLastSyncAndNormalizeBatchID(jobName string) (int64, int64, error) {
```
Comment (iskakaushik) [inline flow/connectors/bigquery/bigquery.go:385]: can we return a struct instead? will be nice to have names.

Comment (serprex) [reply flow/connectors/bigquery/bigquery.go]: Added. Seems a bit wordy when these methods are only called in one place which immediately stores them to a local with a proper name

**Review by iskakaushik — APPROVED** [review]
